### DISCUSSION TITLE
Free student benefits

### DISCUSSION THREAD

**Discussion by olleso1** [community/community#140386] [GitHub Education]
### Hi everyone,

Hello, i got accepted for the free student benefits (copilot etc..) but i dont know/can't see if the benefits has been activated on my profilet yet? It has gone 5 days since i got accepted.

**Comment by davevad93** [comment]
Hi @olleso1 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @olleso1 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
